openwisp_radius/api/serializers.py

```diff
@@ -346,10 +346,23 @@ def validate_phone_number(self, phone_number):
             phone_number = None
         return phone_number
 
+    def validate_user_email(self, email):
+        email = email.lower()
+
+        return email
```

Comment:
just do:

just override `validate_user_email`, try something like:

```python
def validate_email(self, value):
    return super().validate_email(value),lower()
```

The rest should not be necessary.

Please study how field level validation works with DRF.